lib/client.js

```diff
@@ -34,6 +34,18 @@ class Client {
     return await this.s3.copyObject(params).promise();
   }
 
+  async listObjects(params) {
+    return await this.s3.listObjects(params).promise();
```

Comment:
Outside of try/catch blocks, `return await` is redundant. There's even an [ESLint rule](https://github.com/eslint/eslint/blob/master/docs/rules/no-return-await.md) to detect it, but it allows it in try/catch.